Sources/PointFree/MountainNavView.swift

```diff
@@ -8,9 +8,9 @@ import Optics
 import Styleguide
 import Prelude
 
-let mountainNavView = View<(Database.User?, SubscriberState, Route?)> { currentUser, subscriberState, currentRoute in
+let mountainNavView = View<(NavStyle.MountainsStyle, Database.User?, SubscriberState, Route?)> { mountainsStyle, currentUser, subscriberState, currentRoute in
```

Comment:
we thread the mountain nav style through to these views now that we can make some customizations, like the "Point-Free Pointers" logo.